src/kernel/include/pgtbl.h

```diff
@@ -10,7 +10,7 @@
 
 #include "shared/cos_errno.h"
 #include "ertrie.h"
-#include "shared/util.h"
+#include "chal/util.h"
```

Comment:
More importantly, you're completely screwing up the revision history of this file.